pep-9999.rst

```diff
@@ -717,6 +728,30 @@ Maybe::
 Avoid ``Union`` return types, since they require ``isinstance()`` checks.
 Use ``Any`` if necessary.
 
+NamedTuple and TypedDict
+------------------------
+
+Use the class-based syntax for ``typing.NamedTuple`` and
+``typing.TypedDict``, following the Classes section of this style guide.
```

Comment:
I think this should add an internal hyperlink (no RST expert, though):

```suggestion
``typing.TypedDict``, following the Classes_ section of this style guide.
```

Reply:
I need to look into this more. PEP 12 says, `Every unique section title implicitly defines an internal hyperlink target.` We have 2 sections named "Classes", one in the Style Guide and one in Supported Constructs. I rather like the parallelism of having sections named the same, and I can't think of better names. So I'm not sure if it's possible to link to Style Guide.Classes here. :(